### PR TITLE
Renamed transformer to transform

### LINKED ISSUE #30: Change the "transformers" to "transforms"
They're pure functions after all now.

/cc @jdfreder

### PR DESCRIPTION
**Pull request by captainsafia**
I'm breaking into the nteract party!

Closes #30 (maybe). I'm assuming that you meant a purely lexical change in that issue, if that is the case let me know and I will continue making the necessary changes. If not, what be the concern in this most mysterious issue?

cc/ @rgbkrk @jdfreder

### CODE REVIEW

**Comment by rgbkrk** [comment]
The transform functions used to be transformer classes with a transform function. We collapsed them and missed out on re-naming them. We'll need this to be backwards comptaible which should be extra exports of the old names.

**Comment by captainsafia** [comment]
Thanks for clarifying! I'll finish this up within the next couple of hours.

File: test/html.transform.test.js [hunk 1]
```diff
@@ -0,0 +1 @@
+})
```
Comment (jdfreder) [inline test/html.transform.test.js]: Where'd the rest of it go?

Comment (captainsafia) [inline test/html.transform.test.js]: HOW DID I MISS THIS?!!?

Not to self: do not attempt to code at 3am.

**Comment by captainsafia** [comment]
OK. This should be good to merge. Travis is upset due to some configuration issue with zuul but tests on local work.

cc/ @jdfreder @rgbkrk 

Merge me, please.

cute-animals-cat-kitten-begging-pics-2

**Comment by rgbkrk** [comment]
If your tests run fine locally, that's good by me. We'll see how the master merge works out. Let me check the code out.

File: src/transformime.js [hunk 2]
```diff
@@ -182,4 +182,12 @@ class Transformime {
...
 
-export default {Transformime, TextTransformer, ImageTransformer, HTMLTransformer}
+export {
+    Transformime,
+    TextTransform,
+    TextTransform as TextTransformer,
+    ImageTransform,
+    ImageTransform as ImageTransformer,
+    HTMLTransform,
+    HTMLTransform as HTMLTransformer
```
Comment (rgbkrk) [inline src/transformime.js]: Well done!

Comment (rgbkrk) [inline src/transformime.js]: As far as I can tell, the only usage I see of HTMLTransformer is in juptyer-js-output-area. After we make a release with this backwards compatible version, we can fully transition.

**Comment by rgbkrk** [comment]
Here's the one to watch: https://travis-ci.org/nteract/transformime/builds/91896389

**Comment by jdfreder** [comment]
Seems to me like a large part of what makes CI useful is integration into pull request, so authors can see their test results as they go.  @rgbkrk @karissa is zulu is there a way to make zulu authenticate for public PRs too?  Otherwise I think we should investigate alternatives...

**Comment by rgbkrk** [comment]
Pretty annoying that I put the password in plaintext and it's still not happy.

Our other option is cargo culting from the rest of Jupyter and using karma + Firefox on Travis.

**Comment by karissa** [comment]
It's saucelabs.. from this PR it seems like it should still work with plaintext password. https://github.com/travis-ci/travis-ci/issues/1946

**Pull request merged**